packages/sdk/src/api/jsonrpc.ts

```diff
@@ -0,0 +1,119 @@
+import fetch from "cross-fetch";
+
+import { apiConfig } from "../config";
+import { OrditSDKError } from "../errors";
+
+type JsonRpcId = string | number | null;
+
+export type ParamRecord = { [k in string]: Params };
+export type Params =
+  | string
+  | string[]
+  | number
+  | number[]
+  | boolean
+  | boolean[]
+  | null
+  | undefined
+  | ParamRecord;
+
+function isNumber(value: unknown): value is number {
+  return (
+    typeof value === "number" &&
+    value > Number.NEGATIVE_INFINITY &&
+    value < Number.POSITIVE_INFINITY
+  );
+}
+
+function isInteger(value: unknown): value is number {
+  return isNumber(value) && value % 1 === 0;
+}
+
+function isString(value: unknown): value is string {
+  return typeof value === "string";
+}
```

Comment:
Can we move this to utils? Not sure if we will even need this after implementing BigNumber.

Reply:
These are only used internally by `isJsonRpcId`, not used for any amounts